lua/weapons/gmod_tool/stools/wire_dupeport.lua

```diff
@@ -94,14 +98,13 @@ if (SERVER) then
 end
 
 function TOOL:UpdateGhostWireDupePort( ent, player )
-	if ( !ent ) then return end
-	if ( !ent:IsValid() ) then return end
+	if ( not ent ) then return end
+	if ( not ent:IsValid() ) then return end
 
-	local tr 	= util.GetPlayerTrace( player, player:GetAimVector() )
-	local trace 	= util.TraceLine( tr )
-	if (!trace.Hit) then return end
+	local trace = player:GetEyeTrace()
+	if ( not trace.Hit ) then return end
 
-	if (trace.Entity && trace.Entity:GetClass() == "gmod_wire_dupeport" || trace.Entity:IsPlayer()) then
+	if (trace.Entity and trace.Entity:GetClass() == "gmod_wire_dupeport" or trace.Entity:IsPlayer() ) then

```

Comment:
I think this should be
`if (trace.Entity and (trace.Entity:GetClass() == "gmod_wire_dupeport" or trace.Entity:IsPlayer()) ) then`

Reply:
Done, thanks @thegrb93  !

Reply:
I added also ent:IsValid() to avoid getting the class of invalid ents.